README.md

```diff
@@ -5,11 +5,11 @@
 
 API voor het zoeken en raadplegen van gegevens uit de Basisregistratie Kadaster (BRK).
 
-## BRK bevragen v1.4.1 is live!
+## Haal Centraal BRK bevragen v1.5 is live!
 
 BRK bevragen levert nu ook:
-* inzicht in de filiatie van Kadastraal onroerende zaken
-* zoeken in relatie tot gorzen en aanwassen
+* eigenaren van de appartementsrechten die uit erfpacht zijn ontstaan. 
```

Comment:
gesplitste zakelijke rechten en de houders van deze rechten. @kad-pothot is dat correct? Of heb jij een betere formulering?

Reply:
> gesplitste zakelijke rechten en de houders van deze rechten. @kad-pothot is dat correct? Of heb jij een betere formulering?

Nee, dat vindt ik wel een mooie.

Reply:
Aangepast